safe_cli/operators/safe_operator.py

```diff
@@ -811,6 +874,61 @@ def execute_tx(self, safe_tx_hash: Sequence[bytes]) -> bool:
     def get_permitted_signers(self) -> Set[str]:
         return set(self.safe_cli_info.owners)
 
+    def drain(self, to: str):
+        # Getting all events related with ERC20 transfers
+        events = self.ethereum_client.erc20.get_total_transfer_history(
```

Comment:
Did you try this for a real Safe in mainnet or rinkeby? I can guarantee it hangs 😉

Reply:
I tried in rinkeby with 2 tokens but maybe this Safe has few transfers. Should I use another function instead get the total transfer?